### DISCUSSION TITLE
BlockBio

### DISCUSSION THREAD

**Discussion by 0xfynnix** [NonceGeek/indiehacker-handbook#20] [IdeaStack]
将履历信息上链，并且可以得到相关人物公司的链上check，使得简历的信任成本趋近于0

**Comment by leeduckgo** [comment]
> 将履历信息上链，并且可以得到相关人物公司的链上check，使得简历的信任成本趋近于0

是否可以和 BeWater 进行结合？

**Reply by 0xfynnix** [reply]
合理

**Comment by 0xfynnix** [comment]
https://github.com/yangfan3211/block-bio
